R/replace_na.R

```diff
@@ -81,6 +81,6 @@ check_replacement <- function(x, var) {
   n <- vec_size(x)
 
   if (n != 1) {
-    abort(glue("Replacement for `{var}` is length {n}, not length 1."))
+    abort(glue("Replacement for `{var}` is length {n}, not length 1."), call = caller_env())
```

Comment:
Worth adding `error_call = caller_env()` as an argument to `check_replacement()`, even thought it is only used in `replace_na()`.

I think adding the `error_call` argument should be the "go to" solution for most of these, because we never know how the helpers might be used in the future, and having 1 rule to follow means we don't have to think about it as much.

Reply:
Yeah, agreed. I just missed updating these.